### PR TITLE
fix: Desc label with flex node

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/pull/25903#discussion_r511800517

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      Fix Description style when `label` use block element.      |
| 🇨🇳 Chinese |     修复 Description 在 `label` 自定义为块状元素时的样式问题。     |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/descriptions/demo/text.md](https://github.com/ant-design/ant-design/blob/fix-desc/components/descriptions/demo/text.md)

### CODE REVIEW

File: components/descriptions/style/index.less [hunk 1]
```diff
@@ -100,6 +100,11 @@
 
     &-container {
       display: flex;
+
+      .@{descriptions-prefix-cls}-item-label {
```
Comment (yoyo837) [inline components/descriptions/style/index.less:104]: 原来的样式同时会作用于`${itemPrefixCls}-item-label`和`${itemPrefixCls}-item-content`, 这个patch就只恢复了前者

Comment (zombieJ) [reply components/descriptions/style/index.less:104]: 不知道当初 content 是处理什么情况的，我也加一下以防万一好了。

**Review by yoyo837 — APPROVED** [review]

**Comment by paranoidjk** [comment]
https://github.com/ant-design/ant-design/blob/master/components/descriptions/style/index.less#L97
这个老样式不用删除吗？

**Comment by zombieJ** [comment]
嗯，没用了。删

**Pull request merged**